### ISSUE TITLE
[accessibility][voice reader] Voice reader focus is not modal in blocking Dialog (Android)

### ISSUE THREAD

**Issue by lutorre** [OfficeDev/office-ui-fabric-react#10680]
### Environment Information

- **Package version(s)**:  
- **Browser and OS versions**: Google chrome - Android 9 Pie

> Note that our screen reader support is limited to Edge + Narrator. Issues that do not reproduce in Edge + Narrator, and aren't caused by obvious invalid aria values, should be filed with the respective screen reading software, not the Fabric repo.

### Describe the issue:

When a blocking Dialog is displayed while voice over is Enabled the focus doesn't remain modal. 

Steps to reproduce:

1- Activate voice over on an Android device
2- Use chrome browser to open a blocking Dialog 
3- Swipe right until the last component inside the dialog is focused.
4 - Swipe right

   

#### Actual behavior:

The highlighted cursor goes to elements outside the blocking dialog.

#### Expected behavior:

The highlight cursor goes tot he first element within the blocking dialog

**Comment by khmakoto** [comment]
Hi @lutorre, thanks for filing this issue! Can you put up a codepen with your scenario where this repros so that I can take a look at it? If you can't don't worry, I'll spin one up myself and investigate what's happening.

**Comment by khmakoto** [comment]
Hi @lutorre, I've made [this codepen](https://codepen.io/khmakoto/pen/BaaNqqN?editors=0010) following the instructions you posted above and checked what the behavior is with Edge and Narrator. After trying this, I can't seem to repro the bug you're having with VoiceOver on Android. Can you take a look at the codepen, check if it correctly reproes your scenario for you and if that's not the case, then modify the codepen so it reproes your scenario?

Alternatively, if the codepen I've provided does repro your scenario, then, since our screen reader support is limited to Edge + Narrator and it seems that we're using the correct aria values here, my recommendation would be to provide feedback to VoiceOver on Android.

Let me know if there is anything else I can help you with!
